charts/business-api-ecosystem/values.yaml

```diff
@@ -683,6 +683,21 @@ bizEcosystemLogicProxy:
   # -- Base app path of logic proxy (for versions < 8.1.0: /business-ecosystem-logic-proxy, for versions >= 8.1.0: /opt/business-ecosystem-logic-proxy)
   basePath: /business-ecosystem-logic-proxy
 
+  ccs:
+    # -- Endpoint of the CCS, e.g. http://credentials-config-service:8080
+    endpoint: ""
```

Comment:
Maybe this should be commented out to make sure that conditions are not fulfilled by default, like here: https://github.com/i4Trust/helm-charts/blob/main/charts/pdc-portal/values.yaml#L147

Reply:
Empty string has the same effect, and there won't be warnings about non-existing property

Reply:
ok